File: Python/Car.py
# import class ParkingLot file ParkingLot
from ParkingLot import ParkingLot

# deklarasi kelas

class Car (ParkingLot):

    # atribut private
    __jml_kursi = ""
    __jml_pintu = ""

   # konstruktor
    def __init__(self, jml_kursi="", jml_pintu=""):
        super().__init__(jml_kursi, jml_pintu)
        self.__jml_kursi = jml_kursi
        self.__jml_pintu = jml_pintu

    # Getter & Setter
    def get_jml_kursi(self):
        return self.__jml_kursi

    def set_jml_kursi(self, jml_kursi):
        self.__jml_kursi = jml_kursi

    def get_jml_pintu(self):
        return self.__jml_pintu

    def set_jml_pintu(self, jml_pintu):
        self.__jml_pintu = jml_pintu

File: CPP/Car.cpp
// import file header
#include "header.hh"

// konstruktor tanpa parameter
Car::Car()
{
    // set isi atribut secara default
    this->jumlah_kursi = "";
    this->jumlah_pintu = "";
}
// konstruktor dengan parameter
Car::Car(string plat, string merk, string tahun_produksi, string warna, string jumlah_kursi, string jumlah_pintu) : Garage(plat, merk, tahun_produksi, warna, merk_garasi, luas_garasi)
{
    // set isi atribut berdasarkan isi parameter
    this->jumlah_kursi = jumlah_kursi;
    this->jumlah_pintu = jumlah_pintu;
}

/* Getter dan Setter*/

// Get jumlah_kursi
string Car::get_jumlah_kursi()
{
    return this->jumlah_kursi;
}

// Set jumlah_kursi
void Car::set_jumlah_kursi(string jumlah_kursi)
{
    this->jumlah_kursi = jumlah_kursi;
}

// Get jumlah_pintu
string Car::get_jumlah_pintu()
{
    return this->jumlah_pintu;
}

// Set jumlah_pintu
void Car::set_jumlah_pintu(string jumlah_pintu)
{
    this->jumlah_pintu = jumlah_pintu;
}

/* desktruktor */

// default
Car::~Car() // desktruktor
{
}
